### PR TITLE
[DNS] Create tutorial on migration to CF with DNSSEC active

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
PCX-6720

### CODE REVIEW

File: content/dns/dnssec/dnssec-active-migration.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    To add your zone using the API, refer to the [Create Zone endpoint](/api/operations/zones-post).
+
+2. [Review the records found by the automatic scan](/dns/manage-dns-records/how-to/create-dns-records/) or [import your zone file](/dns/manage-dns-records/how-to/import-and-export/).
+
+    To import the zone file using the API, refer to the [Import DNS Records endpoint](/api/operations/dns-records-for-a-zone-import-dns-records).
+
+3. Go to **DNS** > **Settings**, and select **Enable DNSSEC**. Or use the following [API request](/api/operations/dnssec-edit-dnssec-status).
+
+```bash
+curl --request PATCH https://api.cloudflare.com/client/v4/zones/{zone_id}/dnssec \
```
Comment (kodster28) [inline content/dns/dnssec/dnssec-active-migration.md:36]:
```suggestion
curl --request PATCH https://api.cloudflare.com/client/v4/zones/{ZONE_ID}/dnssec \
```

Comment (kodster28) [reply content/dns/dnssec/dnssec-active-migration.md:36]: Should replace throughout with capitalized version.

File: content/dns/dnssec/dnssec-active-migration.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+1. Add Cloudflare DS record to your registrar. You can see your Cloudflare DS record on the [dashboard](https://dash.cloudflare.com/?to=/:account/:zone/dns) by going to **DNS** > **Settings** > **DS Record**.
+2. Add Cloudflare assigned nameservers to your registrar. You can see your Cloudflare nameservers by going to **DNS** > **Records**.
+
+At this point your zone is in a [multi-signer DNSSEC setup](/dns/dnssec/multi-signer-dnssec/).
+
+## 4. Remove previous provider
+
+1. Remove your previous provider's DS record from your registrar.
+2. Remove your previous provider's nameservers from your registrar.
+3. After waiting at least one and a half times the [TTL](https://www.cloudflare.com/learning/cdn/glossary/time-to-live-ttl/) of your previous provider DS record, you can remove the DNSKEY record (containing your previous provider ZSK) that you added to your Cloudflare zone in [step 2](#2-cross-import-zsks).
```
Comment (kodster28) [inline content/dns/dnssec/dnssec-active-migration.md:138]:
```suggestion
3. After waiting at least one and a half times the [TTL](/dns/manage-dns-records/reference/ttl/) of your previous provider DS record, you can remove the DNSKEY record (containing your previous provider ZSK) that you added to your Cloudflare zone in [step 2](#2-cross-import-zsks).
```

**Review by kodster28 — APPROVED** [review]

File: content/dns/dnssec/dnssec-active-migration.md (hunk 2 above)
Comment (RebeccaTamachiro) [reply content/dns/dnssec/dnssec-active-migration.md:138]: I had seen this page in the DNS tile but didn't like the fact that this page goes on to talk about how CF handles TTL depending on Proxy status. Wanted some more generic definition of TTL in this case.

File: content/dns/dnssec/dnssec-active-migration.md (hunk 1 above)
Comment (RebeccaTamachiro) [reply content/dns/dnssec/dnssec-active-migration.md:36]: Hm, from a recent topic in Team Crit, we ended up agreeing to have URL placeholders follow the same format as the API reference docs. Pedro updated in the "Guidelines for curl commands" wiki page as well.

**Pull request merged**